### PR TITLE
golint fixed for routers

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: routers/repo/webhook.go [hunk 1]
```diff
@@ -393,6 +402,7 @@ func TestWebhook(ctx *context.Context) {
 	}
 }
 
+// DeleteWebhook delete a webhook
```
Comment (strk) [inline routers/repo/webhook.go:405]: nitpicking: I think the phrases should be english phrases, like "DeleteWebhook deletes a web hook" (note the ending `s` for third person subject being the function)

**Comment by strk** [comment]
LGTM

**Comment by strk** [comment]
LGTM not working ?

**Comment by thibaultmeyer** [comment]
LGTM

**Comment by lunny** [comment]
resolved #70

**Pull request merged**